docs/mp/introduction.adoc

```diff
@@ -0,0 +1,152 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019, 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Helidon MP
+:description: about Helidon MP
+:keywords: helidon, java, microservices, microprofile
+:rootdir: {docdir}/..
+
+include::{rootdir}/includes/mp.adoc[]
+
+== Introduction
+
+Helidon MP is an Eclipse MicroProfile {version-lib-microprofile-api} runtime that allows the Jakarta EE community to run microservices in a portable way. It designed for ease of use and provides Spring Boot like development experience with heavily usage of dependency injection and annotations. 
+
+== Supported Jakarta EE Specifications
+
+|===
+| Specification | Version | Description
+
+| xref:{rootdir}/mp/beanvalidation.adoc[Jakarta Bean Validation]
+| link:{jakarta-bean-validation-spec-url}[{version-lib-jakarta-bean-validation}]
+| Object level constraint declaration and validation facility
+
+| Jakarta Context and Dependency Injection (CDI)
+| link:{jakarta-cdi-spec-url}[{version-lib-jakarta-cdi}]
+| Declarative dependency injection and supporting services
+
+| Jakarta JSON Processing (JSONP)
```

Comment:
I think we use JSON-P and JSON-B

Reply:
JSONP and JSONB are two different specifications. JSONB is listed in a row below.

Reply:
Ahhh... Fine, I'll change it this way.

Reply:
Done